index.js

```diff
@@ -25,12 +25,13 @@ class MarkdownButtonElement extends HTMLElement {
 
   connectedCallback() {
     if (!this.hasAttribute('tabindex')) {
-      this.setAttribute('tabindex', '0')
+      this.setAttribute('tabindex', '-1')
```

Comment:
This would need to be applied to other `data-md-button` as well, so probably need to be handled in markdown-toolbar.